reference_docs/YAML_models/calconnect_document_attributes.yaml

```diff
@@ -0,0 +1,64 @@
+version: v3.4.5 #Verify versions
+
+inherits_from:
+  # Also may be referred to as “parent flavor”
+  flavor: standoc
+  version: v1.x.x
+
+document_attributes:
+#--- Document attributes  ---#
```

Comment:
But in fact, this is already done: `:docnumber:` is a standoc attribute, with a slight refinement in its description. So there is inheritance, and there is overriding.

The versioning of standoc is redundant, it seems to me: it will be the latest version of standoc consistently, across all flavours that are being maintained. That's what the release process is about.